src/firebase/CRUDServices/experimentServices.js

```diff
@@ -7,17 +7,18 @@ export default class ExperimentServices {
   }
 
     // singleton instance.
-    static sieInstance = null;
+    static experimentServiceInstance = null;
 
     /**
      * creating singleton instance.
      * @return {ExperimentServices} instance
      */
     static getInstance = () => {
-      if (self.sieInstance === null) {
-        self.sieInstance = new ExperimentServices();
+      if (self.experimentServiceInstance === null ||
+          self.experimentServiceInstance === undefined) {
+        self.experimentServiceInstance = new ExperimentServices();
```

Comment:
can we do `if (!self.experimentServiceInstance)` here? I'm just curious

https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in

Reply:
Nit-picky thing: the file names do not match the class names. I.e. `experimentService` vs `ExperimentService`.